### PR TITLE
Element docs update

### PR DESCRIPTION
**Pull request by teslor**

### CODE REVIEW

File: cosmoz-autocomplete.html [hunk 1]
```diff
@@ -1,34 +1,67 @@
...
-
-### Usage
-
-*html example here*
+`<cosmoz-autocomplete>` is an autocomplete/typeahead element.
+
+It has such features as multi selection, autocompletion, validation,
+case-intensive capabilities and some other cool things.
+
+### Examples
```
Comment (nomego) [inline cosmoz-autocomplete.html]: These kind of examples should either be made as inline-demos according to https://www.webcomponents.org/publish (example at https://raw.githubusercontent.com/PolymerElements/paper-button/master/README.md) or moved to the demos. (`demo/`)

File: cosmoz-autocomplete.html [hunk 2]
```diff
@@ -1,34 +1,67 @@
...
 <!--
-`<cosmoz-autocomplete>` is an auto-complete/typeahead multi-select
-
-### Usage
-
-*html example here*
+`<cosmoz-autocomplete>` is an autocomplete/typeahead element.
+
+It has such features as multi selection, autocompletion, validation,
+case-intensive capabilities and some other cool things.
```
Comment (nomego) [inline cosmoz-autocomplete.html]: insensitive.. "some other cool things" ? :)

File: cosmoz-autocomplete.html [hunk 3]
```diff
@@ -1,34 +1,67 @@
...
+
+### Examples
+
+		<cosmoz-autocomplete
+			items="{{ items }}"
+			selectedItem="{{ selectedItem }}"
```
Comment (nomego) [inline cosmoz-autocomplete.html]: this is not a valid attribute name (same for most other examples)

File: cosmoz-autocomplete.html [hunk 4]
```diff
@@ -13,53 +27,27 @@
...
 
-### Examples
-
-		<cosmoz-autocomplete
-			items="{{ items }}"
-			selectedItem="{{ selectedItem }}"
-			placeholder="Search">
-		</cosmoz-autocomplete>
+It provides advanced capabilities such as multi selection, autocompletion, filtering,
+validation, case-intensive capabilities and more!
```
Comment (nomego) [inline cosmoz-autocomplete.html]: case-insensitive

File: README.md [hunk 5]
```diff
@@ -1,22 +1,42 @@
...
+```
 
-`bower install --save Neovici/cosmoz-autocomplete`
+And an array for storing selected items:
+```js
+selected = []
+```
 
-### Add the cosmoz-autocomplete import
+The following code will create `cosmoz-autocomplete` with multiple selection feature:
```
Comment (nomego) [inline README.md]: This should be a webcomponents inline-demo as explained on https://www.webcomponents.org/publish

Comment (teslor) [inline README.md]: Inline-demo is not a mandatory. I think simple example is not worse. Earlier I faced with some issues when using data binding in inline demo, therefore I used static example.

**Comment by teslor** [comment]
I renamed it because it's more standard name for LICENSE file.

**Comment by teslor** [comment]
I added inline demo. It should work. 
However when I'm trying to check it via https://www.webcomponents.org/preview some resources as webcomponents are not loaded. I used the link https://github.com/Neovici/cosmoz-autocomplete/pull/12

**Comment by nomego** [comment]
Yeah the preview seems broken for PRs

**Pull request merged**